src/components/dashboard/AppAccess.vue

```
@@ -9,6 +9,11 @@ import IconLock from '~icons/heroicons/lock-closed'
 import IconPlus from '~icons/heroicons/plus'
 import IconShield from '~icons/heroicons/shield-check'
 import IconTrash from '~icons/heroicons/trash'
+import IconWrench from '~icons/heroicons/wrench'
+import DataTable from '~/components/DataTable.vue'
+import RoleSelect from '~/components/forms/RoleSelect.vue'
+import RoleSelectionModal from '~/components/modals/RoleSelectionModal.vue'
+import SearchInput from '~/components/forms/SearchInput.vue'
```

ESLint

[error] Expected "~/components/forms/SearchInput.vue" to come before "~/components/modals/RoleSelectionModal.vue".

(perfectionist/sort-imports)